etcd/getting-started-with-etcd.md

```diff
@@ -65,9 +65,9 @@ You can also fetch the `docker0` IP programmatically:
 ETCD_ENDPOINT="$(ifconfig docker0 | awk '/\<inet\>/ { print $2}'):2379"
 ```
 
-## Proxy Example
+## Proxy example
 
-Let's pretend we're setting up a service that consists of a few containers that are behind a proxy container. We can use etcd to announce these containers when they start by creating a directory, having each container write a key within that directory and have the proxy watch the entire directory. We're going to skip creating the containers here but the [docker guide]({{site.baseurl}}/os/docs/latest/getting-started-with-docker.html) is a good place to start for that.
+Let's pretend we're setting up a service that consists of a few containers that are behind a proxy container. We can use etcd to announce these containers when they start by creating a directory, having each container write a key within that directory and have the proxy watch the entire directory. We're going to skip creating the containers here but the [docker guide](../os/getting-started-with-docker.md) is a good place to start for that.
```

Comment:
To the best of my prior knowledge, relative links within a directory structure work. It is links that cross repositories that require special consideration. I think this link is OK.